src/main/java/domain/LottoPrize.java

```diff
@@ -4,23 +4,21 @@
 import java.util.Comparator;
 
 public enum LottoPrize {
-    FIRST(2000000000, 6, 0, 1),
-    SECOND(30000000, 5, 1, 2),
-    THIRD(1500000, 5, 0, 3),
-    FOURTH(50000, 4, 0, 4),
-    FIFTH(5000, 3, 0, 5),
-    NOTHING(0, 0, 0, Integer.MAX_VALUE);
+    FIRST(2000000000, 6, 0),

```

Comment:
```suggestion
    FIRST(2_000_000_000, 6, 0),
```
가독성을 높이기 위해 이렇게 사용하곤 해요!

Reply:
꿀팁 감사합니다!